front/lib/api/assistant/agent_usage.ts

```diff
@@ -1,58 +1,66 @@
-import type { AgentUsageType, ModelId } from "@dust-tt/types";
+import type {
+  AgentUsageType,
+  LightAgentConfigurationType,
+  ModelId,
+  WorkspaceType,
+} from "@dust-tt/types";
 import { literal, Op } from "sequelize";
 import { v4 as uuidv4 } from "uuid";
 
+import { getUsersWithAgentInListCount } from "@app/lib/api/assistant/user_relation";
+import { getMembersCount } from "@app/lib/api/workspace";
+import type { Authenticator } from "@app/lib/auth";
 import {
   Conversation as DBConversation,
   Mention,
   Message,
   UserMessage,
   Workspace,
 } from "@app/lib/models";
+import { AgentUserRelation } from "@app/lib/models/assistant/agent";
 import { redisClient } from "@app/lib/redis";
 
 // Ranking of agents is done over a 30 days period.
 const rankingTimeframeSec = 60 * 60 * 24 * 30; // 30 days
 
 function _getKeys({
-  workspaceId,
-  agentConfigurationId,
+  workspaceSId,
```

Comment:
Renamed to `xxxSId` as this code lives in the front and it's pretty confusing with the regular id.

Reply:
I thought this was true for connectors where we only deal with `sId`s. Here it's pretty confusing to have `workspaceId` as a string and on the same method `messageId` as a number.

Reply:
Let’s call it messageModelId 👍